flang/include/flang/Optimizer/Transforms/Passes.td

```diff
@@ -197,4 +197,12 @@ def ExternalNameConversion : Pass<"external-name-interop", "mlir::ModuleOp"> {
   ];
 }
 
+def SimplifyRegionLite : Pass<"simplify-region-lite", "mlir::ModuleOp"> {
+  let summary = "Region simplification";
+  let description = [{
+    Run region DCE and erase unreachable blocks on regions.
```

Comment:
Should this read "blocks in regions" rather than "blocks on regions"?

Reply:
Yes.